confirm-deposit.md

```diff
@@ -0,0 +1,120 @@
+# Confirm Deposits to the Exchange
+
+Deposits are made in several stages. 
+After the user deposits funds to their assigned reception wallet (step 1 in [Detecting Deposits](https://docs.flare.network/exchange/architecture/#detecting-deposits)), it goes through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a [blockchain](https://docs.flare.network/tech/glossary/), multiple [validators](https://docs.flare.network/tech/validators/) examine newly created transactions and decide on whether they are valid and which [block](https://docs.flare.network/tech/glossary/) to add to the chain next. 
+It takes time for all the validators to agree.
+Until they reach agreement, validators may add or revert transactions to a block. 
+After the validation process is complete, blocks become irreversible. 
```

Comment:
Here's what I'm thinking here:

It seems to me that it’s a common understanding that blocks on the blockchain are immutable. 

Question: In the case of validation, is their mutability a function of WHO (validators can revert, non-validator-attackers can't?), or a function of WHEN (once all/most validators agree or a certain state of the consensus algorithm is reached?)? 

I think it's important to clarify, bc at some point, we'll probably need to invoke block immutability and they may be confused.